File: perl5/trivial.pl
#!/usr/bin/perl

# perl, my very recently discovered attraction

use strict;
use warnings;

for my $i (1 .. 100) {
  my $str;
  $str .= "Fizz" unless $i % 3;
  $str .= "Buzz" unless $i % 5;
  printf "%s\n", $str // $i;
}

File: es6/trivial.js
#!/usr/bin/node --harmony

// when i started working with JS, Firefox was known
// as Phoenix, and `document.write` ruled the world.
// it's very different now, more than the splat in this
// implementation would admit...

let print = console.log

;;;

[...Array(101).keys()].slice(1).forEach((i) => {
       if (i % 15 == 0) print("FizzBuzz")
  else if (i %  3 == 0) print("Fizz")
  else if (i %  5 == 0) print("Buzz")
  else                  print(i)
})
